### PR TITLE
Connect only once

### PR DESCRIPTION
**Pull request by multi**
The idea of this PR is to fix an issue while running two cells from one process (eg. cells started from mocha) to call `.connect()` only once, otherwise there is an error thrown (Trying to open unclosed connection)

### CODE REVIEW

**Comment by outbounder** [comment]
@multi it is cool PR but I cant merge it directly because I think this feature/behavior should be behind a flag such as `dna.reuseMongooseConnection` or similar. 

The reason behind this is that usually if one is using `organic-mongoose` organelle and it should react its reaction should be straightforward namely to call mongoose.connect and if there is already a connection present then the organelle shouldn't be used at all OR it should be used with explicitly stating that existing mongoose connection if present should be used. 

Otherwise the feature presented with the PR in some situations could lead to problems such as thinking that having two instances of `organic-mongoose` organelle means that there are two connections to mongodb through mongoose :)

----

Bottom line is - do you think this makes sense and are you willing to update the PR accordingly? :)

**Comment by multi** [comment]
@outbounder just updated the PR accordingly 👍

**Comment by outbounder** [comment]
@multi thanks. Released in organic-mongoose@1.1.0
